GitUI/GitModuleForm.cs

```diff
@@ -59,17 +59,12 @@ public GitModuleForm(bool enablePositionRestore, GitUICommands commands)
 
         protected override bool ExecuteCommand(int command)
         {
-            if (ExecuteScriptCommand(command))
+            if (Script.ScriptRunner.ExecuteScriptCommand(this, Module, command))
```

Comment:
[nit] could be simplified further to `return Script.ScriptRunner... || base.ExecuteCommand`

Reply:
Sorry, I missed this. Fixed in 8e56bb44c.